CHANGELOG.next.md

```diff
@@ -30,6 +30,8 @@ Thanks, you're awesome :-) -->
 
 ### Tooling and Artifact Changes
 
+* Generate include files to ensure referenced Github branch matches docs #1338
```

Comment:
> @ebeahan as this is following the Feature Freeze, I'm getting a conflict with CHANGELOG.next.md and could appreciate guidance there

Yeah, that can happen if the `CHANGELOG.next` file is updated while you're working on your branch here.

Looking at the changelog entry here, we actually want to place it under one of the section headers underneath `Tooling and Artficact Changes`. Perhaps `Improvements`? Moving the entry might also fix the conflict.

Reply:
OK, I moved it back there :+1: